### DISCUSSION TITLE
Min.pct in FindMarkers for motif DE analysis

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [stuart-lab/signac#1144] [Q&A] [answered]
Hi, I want to check the differential enrichment of motif before and after a treatment using the `FindMarkers()` function. However, I get the error:

`no features pass min.pct threshold returning empty data.frame`

In terms of an rna assay, I realize that it excludes the features that are present (score >0) for at least certain percent based on the number we passed. What does it mean in terms of chromvar assay? Does it still only consider the motifs with z-score greater than 0 or is it abs(zscore) > 0? 

- When I looked at the deviation matrix, there were so many NA values. They seem to be causing the error. What approach would you take based to exclude the na values? Is there a way to not filter based on min.pct altogether?

**Comment by timoast** [comment, marked as answer]
When running `FindMarkers()` on z-score data you need to set `mean.fxn=rowMeans` so that the fold-change calculation computes the average difference in z-score between the groups (shown in the [motif](https://satijalab.org/signac/articles/motif_vignette.html) vignette), and you can set `min.pct=0` to ensure all the features are tested (since "percent expressing" doesn't make sense for centered data).
